### DISCUSSION TITLE
v1.6.0-beta.2

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#150] [Announcements]
- New Device: Adds support for the following iCUE LINK devices: XC7 RGB ELITE (Stealth Gray), XC7 RGB ELITE (White); (#139).
- iCUE LINK: Minimum pump power now configurable using the `FANCONTROL_CORSAIRLINK_MIN_PUMP_DUTY` environment variable.
- Commander CORE: Minimum pump power now configurable using the `FANCONTROL_CORSAIRLINK_MIN_PUMP_DUTY` environment variable (#149).
- Internals: Fixed the CorsairLink.log file being opened for writing unnecessarily (#147).

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.6.0-beta.2'>v1.6.0-beta.2</a>.</em>

**Comment by bjongbloed** [comment]
It's working for me, thank you! (#149)
